src/style.css

```diff
@@ -0,0 +1,143 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: "Open Sans", sans-serif;
+  font-weight: 300;
+}
+
+.search-container {
+  display: flex;
+  justify-content: center;
+  align-items: left;
+  margin-top: 10px;
+  margin-left: 12px;
+}
+
+form {
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+  height: 48px;
+  padding: 0 16px;
+}
+
+input[type="text"] {
```

Comment:
The same for this selector

Reply:
Don't use tag selectors. Use class selectors instead.
You can add many classes to one element